transformer_engine/pytorch/module/linear.py

```diff
@@ -72,8 +74,9 @@ def forward(
         input_quantizer: Optional[Quantizer],
         weight_quantizer: Optional[Quantizer],
         output_quantizer: Optional[Quantizer],
-        grad_output_quantizer: Optional[Quantizer],
-        grad_input_quantizer: Optional[Quantizer],
+        gradient_quantizer: Optional[Quantizer],
+        dgrad_quantizer: Optional[Quantizer],
+        wgrad_quantizer: Optional[Quantizer],
```

Comment:
I think these name changes make things less clear and they break the naming symmetry:
- `input` : `grad_input`
- `output` : `grad_output`
- `weight` : `grad_weight`

`gradient` is especially unclear, since `wgrad` and `dgrad` are also gradients.